src/software/jetson_nano/services/network.cpp

```diff
@@ -36,21 +33,36 @@ std::tuple<TbotsProto::PrimitiveSet, TbotsProto::World> NetworkService::poll(
 void NetworkService::primitiveSetCallback(TbotsProto::PrimitiveSet input)
 {
     std::scoped_lock<std::mutex> lock(primitive_set_mutex);
-    primitive_set_msg = input;
+    primitive_set_msg   = input;
+    const auto& seq_num = input.sequence_number();
```

Comment:
For better readability, we should avoid using the `auto` keyword too much unless the type has a really long name.